code/game/objects/items/weapons/grenades/explosive.dm

```diff
@@ -21,4 +21,13 @@
     qdel(src)
 
 /obj/item/weapon/grenade/explosive/proc/on_explosion(var/turf/O)
-	explosion(O, devastation_range, heavy_range, weak_range, flash_range)
\ No newline at end of file
+	explosion(O, devastation_range, heavy_range, weak_range, flash_range)
+
+/obj/item/weapon/grenade/explosive/nt
+	name = "NT OBG \"Holy Grail\""
+	desc = "A military-grade offensive blast grenade, designed to be thrown by assaulting troops."
```

Comment:
Holy Hand Grenade.
```suggestion
	desc = "There's an inscription along the bands. \'And the Lord spake, saying, 'First shalt thou take out the Holy Pin. Then, shalt thou count to three, no more, no less. Three shall be the number thou shalt count, and the number of the counting shall be three. Four shalt thou not count, nor either count thou two, excepting that thou then proceed to three. Five is right out! Once the number three, being the third number, be reached, then lobbest thou thy Holy Hand Grenade of Antioch towards thou foe, who being naughty in my sight, shall snuff it.'\"
```

(This is entirely optional and up to personal preference)

Reply:
Done